signalfx-java/src/main/java/com/signalfx/metrics/connection/AbstractHttpDataPointProtobufReceiverConnection.java

```diff
@@ -15,6 +20,7 @@
 import com.signalfx.endpoint.SignalFxReceiverEndpoint;
 import com.signalfx.metrics.SignalFxMetricsException;
 import com.signalfx.metrics.protobuf.SignalFxProtocolBuffers;
+import org.apache.http.message.BasicNameValuePair;
```

Comment:
Import order is wrong

Reply:
Oops, I just assumed IntelliJ would do them right and never looked. I cleaned up some unused imports while I was here.